swig/swigResqml2Include.i

```diff
@@ -4823,6 +4842,7 @@ namespace RESQML2_NS
 	{
 	public:		
 		unsigned int getWellboreMarkerCount();
+		std::vector<RESQML2_NS::WellboreMarker *> getWellboreMarkerSet() const;
```

Comment:
Should propbably define and use macro for getters.

Reply:
Done.